app/schemas/project_preference_update_project_settings_schema.rb

```diff
@@ -0,0 +1,21 @@
+class ProjectPreferenceUpdateSchema < JsonSchema
```

Comment:
this will need the DSL config stuff for the json_schema validator we talked about.